packages/comet-admin-rte/src/core/types.ts

```diff
@@ -46,3 +46,18 @@ export interface IControlProps {
 export type ToolbarButtonComponent = (props: IControlProps) => JSX.Element;
 
 export type FilterEditorStateFn = (nextState: EditorState) => EditorState;
+
+/**
+ * @deprecated use IBlocktypeConfig instead
+ */
+interface ICustomBlockType_Deprecated {
```

Comment:
Those interfaces were not part of the public API (no export), so we can safely rename them.